CourseApp/ClassTaskFolder/DocumentValue.cs

```diff
@@ -0,0 +1,103 @@
+namespace CourseApp.Class
+{
+    using System;
+    using static System.Math;
+
+    public class DocumentValue
+    {
+        public static int IntValue()
+        {
+            bool isNumber = false;
+            var input = string.Empty;
+            while (!isNumber)
+            {
+                input = Console.ReadLine();
+                isNumber = int.TryParse(input, out int inputValue);
+                if (!isNumber)
+                {
+                    Console.WriteLine("Please enter correct value");
+                }
+            }
+
+            return Convert.ToInt32(input);
```

Comment:
We already have `inputValue` that has type `int`, we need not to convert input

Reply:
Fixed